internal/service/advancedcluster/schema.go

```diff
@@ -313,6 +313,10 @@ func resourceSchema(ctx context.Context) schema.Schema {
 				Update: true,
 				Delete: true,
 			}),
+			"use_effective_fields": schema.BoolAttribute{
+				Optional:            true,
+				MarkdownDescription: "Flag that indicates whether to use the effective fields for the cluster.",
```

Comment:
```suggestion
				MarkdownDescription: "Controls how hardware specification fields are returned in the response. When set to true, returns the original client-specified values and provides separate effective fields showing current operational values. When false (default), hardware specification fields show current operational values directly. Primarily used for autoscaling compatibility.",
```

I'd use the same description as in API: https://www.mongodb.com/docs/api/doc/atlas-admin-api-v2/operation/operation-getgroupcluster#operation-getgroupcluster-headers

also if we agree on the description, for example this proposed description, I can use the same in the documentation PR

Reply:
Yep, SGTM!

Reply:
Addressed in [7b5a7f5](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/3895/commits/7b5a7f540f9ff61c92f5a36efa9377bfe17da10f)